lib/brain.js

```diff
@@ -2945,6 +2946,13 @@ Brain.prototype._reboot = function _reboot () {
   })
 }
 
+Brain.prototype._shutdown = function _shutdown () {
+  console.log('Remote shutdown')
+  cp.execFile('shutdown', ['now'], {}, function () {
+    process.exit(0)
```

Comment:
Shouldn't we attempt a more graceful shutdown, like only when the machine is idle?

Reply:
Decided to do this is an separate card.